pyrasterframes/src/main/python/docs/raster-read.pymd

```diff
@@ -55,6 +53,64 @@ tile = rf.select(rf_tile("proj_raster")).first()[0]
 display(tile)
 ```
 
+## Multiple Singleband Rasters
+
+In this example, we show reading [two bands](https://en.wikipedia.org/wiki/Multispectral_image) of [Landsat 8](https://landsat.gsfc.nasa.gov/landsat-8/) imagery (red and near-infrared), combining them with `rf_normalized_difference` to compute NDVI. As described in the section on @ref:[catalogs](raster-catalogs.md), image URIs in a single row are assumed to be from the same scene/granule, and therefore compatible. This pattern is commonly used when multiple bands are stored in separate files.
```

Comment:
Possibly link internal to concepts.html#band ?

Reply:
And also maybe link to some external resource about NDVI.  Consider a parenthetical like, `[NDVI](some-link.html), a common measure of plant vigor.`.